sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -275,9 +275,9 @@ revamped configuration API. Static `create_config` methods have been renamed
 
 ### New features:
 - Added credentials for authenticating users:
-[`DeviceCodeCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.DeviceCodeCredential),
-[`InteractiveBrowserCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.InteractiveBrowserCredential),
-[`UsernamePasswordCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.UsernamePasswordCredential)
+[`DeviceCodeCredential`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.2.0/azure.identity.html#azure.identity.DeviceCodeCredential),
```

Comment:
This is the changelog for the pre-GA beta 2. We seem not to have docs for it anymore, which I think is okay because it's a year old and unsupported. So although the current links 404, is linking to documentation of a later, significantly different release really an improvement?

Reply:
Is it ok if i remove these links entirely?

Reply:
I have removed the links from these